### ISSUE TITLE
Slack

### ISSUE THREAD

**Issue by choldgraf** [voxverus/JobTalk#6]
what's the best way to join the slack team? Open an issue like this one? I'm curious to see what conversation is happening around this idea, I think it's great.

**Comment by cxrodgers** [comment]
Agreed! I have general questions about overall approach that may best be resolved in a group conversation before creating/working on issues. I clicked the "Join our slack team" link on the github readme but slack rejected my password so either I am doing something wrong or the team is not set up to accept randos from the internet

**Comment by choldgraf** [comment]
@cxrodgers by default slack won't let you just sign up on your own, you have to be invited. It is pretty easy way to set up a heroku app that will auto-signup people. I can help set that up if it's of interest

**Comment by aromartins** [comment]
Hi guys, thanks for stopping by! Yes indeed slack does not seem to allow self-registration. 
Dabbled with this little heroku app - https://jobtalk-self-sign-in.herokuapp.com/   
Give it a try, let me know if it doesn't work! 
Looking forward to brainstorming :)
